fossology/fossology
# feat(php): Replaced the class Object by builtin features.

## Description

Since PHP 7.2 `Object` is a reserved word. Instead of renaming our class Object we can replace it by builtin features of php.

### Changes

* A lot of tests.
* `lib/php/Dao` used the static method `classname()` to instantiating its logging.
* `lib/php/View/HighlightProcessor.php`

## How to test

* Run the tests or check the logs on travis-ci.
* Check logging is working for `lib/php/Dao`.

Thanks to @GMishx for pointing out this backwards compatibility issue in #1116.
